### DISCUSSION TITLE
Apache Oltu is a deprecated project

### DISCUSSION THREAD

**Discussion by d3bt3ch** [ory/.github#34] [Ideas]
Apache Oltu is a deprecated (attic) project. Can we update the dependency to some other library? I can help refactoring the SDK.

**Comment by vinckr** [comment]
Hey @debjitk 
That would be good! 
Could you open an issue here (https://github.com/ory/sdk/issues) if you already have an idea with what to replace it?

**Reply by d3bt3ch** [reply]
Sure

**Reply by vinckr** [reply]
Much appreciated!
